packages/cadl-ranch-dashboard/src/apis.ts

```diff
@@ -6,11 +6,12 @@ import {
 
 const storageAccountName = "azuresdkcadlranch";
 
-export type GeneratorNames = "python" | "typescript" | "csharp" | "java" | "test";
+export type GeneratorNames = "python" | "typescript-rlc" | "typescript-modular" | "csharp" | "java" | "test";
```

Comment:
```suggestion
export type GeneratorNames = "python" | "typescript/rlc" | "typescript/modular" | "csharp" | "java" | "test";
```

could we use `/` as a seprator that way we can use that to group things in the UI at a later point